subjects/rockandroll/README.md

```diff
@@ -0,0 +1,45 @@
+## rockandroll
+
+### Instructions
+
+Write a function called `RockAndRoll` that takes an `int` and returns a `string`.
+
+- If the number is divisible by 2, print `rock` followed by a newline `\n`.
+- If the number is divisible by 3, print `roll` followed by a newline `\n`.
+- If the number is divisible by 2 and 3, print `rock and roll` followed by a newline `\n`.
+
+### Expected function
+
+```go
+func RockAndRoll(n int) string {
+
+}
+```
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	fmt.Println(RockAndRoll(4))
+	fmt.Println(RockAndRoll(9))
+	fmt.Println(RockAndRoll(6))
```

Comment:
Maybe add here edge cases or simply mentioning them above is enough. For instance:
fmt.Println(RockAndRoll(-10)) = ? what should it print
fmt.Println(RockAndRoll(11)) = ?

Reply:
Same logic as the above answer!

Reply:
Same logic as the answer above.